src/localsocketcontroller.cpp

```diff
@@ -67,6 +77,13 @@ void LocalSocketController::initialize(const Device* device, const Keys* keys) {
 
 #ifdef MVPN_WINDOWS
   QString path = "\\\\.\\pipe\\mozillavpn";
+  if(!m_serviceManager.isRunning()){
+    logger.log() << "Deamon is not running, wait until deamon is ready";
+    // We will retry once the deamon is ready
+    m_serviceManager.startService();
```

Comment:
1. startService is confusing because the controller cannot start the service. Probably we need a better name.
2. Should we simply communicate the failure and wait for the user to click the 'restore' button? Asking this because, let's say that the service is unable to start (a crash at boot, or anything similar), the UI will be kind-of frozen. The user in the initializing state cannot do any useful operation with the VPN.
thoughts?

Reply:
1. Maybe startDeamon then? 
2. I see the edge case, but i dont think !isRunning() is inheritly a failure, given that we're in initialize(). Maybe a serviceStartFailure Signal, which we then escalate to the user with the restore button?

Reply:
i added more errorHandles() so in case we're failing this, we can handle it.